lib/order.rb

```diff
@@ -7,12 +35,78 @@ def initialize(id, products)
       @products = products
     end
 
+    def self.all
+      orders_array = []
+      ind_order_array = []
+
```

Comment:
Good variable name on `orders_array`. I don't know that `ind_order_array` is ever used - looks like you had some debugging around it below, but it's currently commented out.